packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
+import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
+
+const StyledEmbedError = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_BODY_COPY};
+  background-color: ${C_CHALK};
+  background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 38.2%;
+  color: ${C_EBON};
+  height: 100vh;
```

Comment:
Looks good 👍 

Would be nice if we could get this to not always assume it is going to be in an iframe though - the 100vh will look weird in any other scenario.  If we know the media will fail, we'd like to be able to circumvent the embed endpoint entirely.

This is probably more relevant to media pod pages than articles though, so happy to handle this later

Reply:
Thanks, Ryan, this is a good suggestion.

I've implemented a `fillViewport` prop, which, when active, will cause this component to fill the viewport. By default this component now fills its parent container.